tests/unit_tests/udf_wrapper_params/zero_shot/multiple_labels_single_model_single_batch.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import PurePosixPath
+from exasol_udf_mock_python.connection import Connection
+from tests.unit_tests.udf_wrapper_params.zero_shot.mock_zero_shot import \
+    MockZeroShotFactory, MockZeroShotModel, MockPipeline
+
+
+def udf_wrapper():
+    from exasol_udf_mock_python.udf_context import UDFContext
+    from exasol_transformers_extension.udfs.models.\
+        zero_shot_text_classification_udf import ZeroShotTextClassificationUDF
+    from tests.unit_tests.udf_wrapper_params.zero_shot.mock_sequence_tokenizer \
+        import MockSequenceTokenizer
+    from tests.unit_tests.udf_wrapper_params.zero_shot.\
+        multiple_labels_single_model_single_batch import \
+        MultipleLabelsSingleModelSingleBatch as params
+
+    udf = ZeroShotTextClassificationUDF(
+        exa,
+        batch_size=params.batch_size,
+        pipeline=params.mock_pipeline,
+        base_model=params.mock_factory,
+        tokenizer=MockSequenceTokenizer)
+
+    def run(ctx: UDFContext):
+        udf.run(ctx)
+
+
+class MultipleLabelsSingleModelSingleBatch:
+    """
+    multiple labels, single model, single batch, batch complete
+    """
+    batch_size = 1
+    data_size = 1
+
+    input_data = [(None, "bfs_conn1", "sub_dir1", "model1",
```

Comment:
test also multiple labels and topk>1

Reply:
top_k is not a parameter for this model. This model produces predictions for each label.

Reply:
ok, then I don't understand this part https://github.com/exasol/transformers-extension/blob/1306c2d5755e1a144a067568a5c9f8c61cbb6726/exasol_transformers_extension/udfs/models/zero_shot_text_classification_udf.py#L35

Reply:
wrong naming :+1:

Reply:
Please, fix the naming, because it is confusing

Reply:
I'll do, working on size issue